var/www/html/wp-config.php.ctmpl

```diff
@@ -19,6 +19,12 @@ define( 'DB_PASSWORD', '{{env "MYSQL_PASSWORD"}}' );
 define( 'DB_HOST',     '{{if service "mysql-primary"}}{{with index (service "mysql-primary") 0}}{{.Address}}:{{.Port}}{{end}}{{end}}' );
 
 /**
+ * Handle SSL reverse proxy
```

Comment:
See https://github.com/autopilotpattern/wordpress/pull/34/files#r85470880 for how to eliminate this conditional